cmd/sheltertech-go/integration_test.go

```diff
@@ -61,34 +61,70 @@ func TestGetCategories(t *testing.T) {
 	assert.Equal(t, "12-Step", categoriesResponse[0].Name, "12-step is the first category ordered by Name")
```

Comment:
Can't comment on the line just above this, since it's just outside of the range of the diff, but did you want to change these two assertions to not be dependent on the specific values in the DB? Your changes to the other tests included checking that the `Id` and `Name` fields are not empty, but these assertions are still assuming a particular state of the DB.

Reply:
I'd generally agree that we probably should be careful about tightly coupling to any test data at this point.